Assets/VRTK/Scripts/VRTK_Hip_Tracking.cs

```diff
@@ -0,0 +1,73 @@
+﻿namespace VRTK
+{
+	using UnityEngine;
+	using System.Collections;
+
+	public class VRTK_Hip_Tracking : MonoBehaviour {
+
+		/// <summary>
+    	/// Hip Tracking attempts to reasonably track hip position in the absence of a hip position sensor
+    	/// </summary>
+   	 	/// <remarks>
+		///
+		/// The Hip Tracking script is placed on an empty GameObject which will be positioned at the estimated hip position
+		/// <para>&#160;</para>
+		/// Variables
+		/// <list type="bullet">
+    	/// <item>Head Offset<description>Distance underneath Player Head for hips to reside.</description> </item>
+		/// <item>Head Override<description>Optional Transform to use as the Head Object for calculating hip position. If none is given one will try to be found in the scene.</description> </item>
```

Comment:
I dont think these `description` tags would work. Check out the AdaptiveQuality script to see how the lists were done in those comments